### ISSUE TITLE
FBU Interface

### ISSUE THREAD

**Issue by demarley** [cms-ttbarAC/CyMiniAna#17]
Interface with [FBU](https://github.com/gerbaudo/fbu) 
via [uproot](https://github.com/scikit-hep/uproot) to do unfolding.  

Using uproot to load flat ntuples (without ROOT) and easily have access to the information for FBU.

**Comment by demarley** [comment]
Create simple unfolding class that interfaces with FBU.

**Comment by demarley** [comment]
On hold until the CWoLa results can be prepared

**Comment by demarley** [comment]
Now that the HATS is complete, this can be revisited

**Comment by demarley** [comment]
Jupyter notebooks have been created that can perform the unfolding (for simplified cases).

**Comment by demarley** [comment]
Mostly addressed now.  Waiting on systematics in the analysis before more can be done -- closing now.
